.github/CODEOWNERS

```
@@ -12,6 +12,7 @@
 /manifests/cpp.yml @DataDog/system-tests-core
 /manifests/dotnet.yml @DataDog/apm-dotnet @DataDog/asm-dotnet @DataDog/system-tests-core
 /manifests/golang.yml @DataDog/apm-go @DataDog/system-tests-core
+/manifests/java.yml @DataDog/apm-java @DataDog/system-tests-core
```

```suggestion
/manifests/java.yml @DataDog/asm-java @DataDog/apm-java @DataDog/system-tests-core
```

@DataDog/asm-java is the team using these tests the most, and we're not part of `@DataDog/apm-java`.